apps/server/src/modules/fwu/controller/fwu.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Controller, Get, Req, Res, InternalServerErrorException } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { Authenticate } from '@src/modules/authentication/decorator/auth.decorator';
+import { Request, Response } from 'express';
+import { Configuration } from '@hpi-schul-cloud/commons';
+import { FwuUc } from '../uc/fwu.uc';
+
+@ApiTags('FWU')
+@Authenticate('jwt')
+@Controller('fwu')
```

Comment:
Why not "wtf"?
Please use clarify naming for module/us/conroller and routes

Reply:
Would "fwu-content" be clearer? 
Otherwise, we are open to suggestions.

Reply:
Yes, that's better, and I think the name of the module can just be "learning content". If we have many types of content

Reply:
Because we only cover FWU content and no other learning content, we would then change the name to "fwu-learning-contents".